docs/markdown/Contributing.md

```diff
@@ -264,15 +264,18 @@ current platform. The following values are currently supported:
 | type          | Description                                                                                             |
 | ------------- | ------------------------------------------------------------------------------------------------------- |
 | `file`        | No postprocessing, just use the provided path                                                           |
+| `python_file` | Use the provided path while replacing the python directory.                                             |
 | `dir`         | To include all files inside the directory (for generated docs, etc). The path must be a valid directory |
 | `exe`         | For executables. On Windows the `.exe` suffix is added to the path in `file`                            |
 | `shared_lib`  | For shared libraries, always written as `name`. The appropriate suffix and prefix are added by platform |
+| `python_lib`  | For python libraries, always written as `python_file`. The appropriate suffix is added by platform      |
```

Comment:
shoudl'nt this say `python_lib` in the Description?

Reply:
Well, I meant it follows the same format as python_file. idk, describing things here almost takes more time than writing the actual code...